app/assets/javascripts/discourse/components/emoji-uploader.js

```diff
@@ -1,31 +1,61 @@
 import { notEmpty, not } from "@ember/object/computed";
+import { action } from "@ember/object";
 import Component from "@ember/component";
-import discourseComputed from "discourse-common/utils/decorators";
+import { default as discourseComputed } from "discourse-common/utils/decorators";
```

Comment:
We'll need to agree on a single style here. 😅 Because I've been doing the opposite:

```js
//  Replacing e.g.
import { default as computed, other } from "x";
// with
import computed, { other } from "x";
```